src/modules/auth/index.js

```diff
@@ -218,16 +218,11 @@ export default class AuthModule extends Module {
 
 
 	async _isUserAuthorized() {
-		let resp = false;
+		let authorized = false;
 		const state = this.Store.getState();
 		let activeTenant = state.tenant.current;
```

Comment:
This method needs to be implemented as two-variants:
1) with tenant service  
2) without tenant service

Reply:
Refactored in this commit: https://github.com/TeskaLabs/asab-webui/pull/105/commits/1732113a864b62f879e02d01aa5aed2eec6a7698

Added description in this commit: https://github.com/TeskaLabs/asab-webui/pull/105/commits/c131be0d0b0d325488737d4f0b11de06b0b13287